### PR TITLE
feat(editor): set sane defaults

### PR DESCRIPTION
**Pull request by williammartin**

### CODE REVIEW

File: src/views/Studio/index.vue [hunk 1]
```diff
@@ -140,8 +140,12 @@ export default class Studio extends Vue {
   private options: any = {
     readOnly: true,
     minimap: { enabled: false },
-    fontSize: 16,
-    automaticLayout: true
+    fontSize: 12,
+    automaticLayout: true,
```
Comment (matthewhudson) [inline src/views/Studio/index.vue]: 👍👍👍 `automaticLayout`

**Comment by williammartin** [comment]
@Arinono What's up with the pending statuses here? Are they stuck?

**Comment by Arinono** [comment]
@williammartin GitHub Actions seem to be blocked [here](https://github.com/storyscript/studio/actions?page=2). I canceled the last workflow, it was still running even if all the test suites were done

**Comment by williammartin** [comment]
Can we merge?

**Comment by Arinono** [comment]
Yes, this is safe changes 😉

**Pull request merged**